ui/src/pageLayout/components/OrgSwitcherItem.tsx

```diff
@@ -0,0 +1,51 @@
+// Libraries
+import React, {FC} from 'react'
+import classnames from 'classnames'
+import {WithRouterProps, withRouter} from 'react-router'
+
+// Components
+import {IconFont, Icon} from '@influxdata/clockface'
+
+interface ComponentProps {
+  orgName: string
+  orgID: string
+  selected: boolean
+  onDismiss: () => void
+}
+
+type Props = ComponentProps & WithRouterProps
+
+const OrgSwitcherItem: FC<Props> = ({
+  orgName,
+  selected,
+  onDismiss,
+  orgID,
+  router,
+}) => {
+  const orgSwitcherItemClass = classnames('org-switcher--item', {
+    'org-switcher--item__selected': selected,
+  })
+
+  const orgSwitcherIcon = selected ? IconFont.Checkmark : IconFont.CaretRight
+
+  const handleClick = (): void => {
+    onDismiss()
```

Comment:
I imagine that we'll pull new data based on the new orgID, but it might be worth checking to see if we'd need to reset the app state for particular parts of the app